### PR TITLE
Update ncov_sit-rep_2020-03-02.md

### PR DESCRIPTION
**Pull request by misjailcisin**
Addition of COVID-19 FAQ section and a few edits throughout to correctly update case counts, etc.

### CODE REVIEW

File: narratives/ncov_sit-rep_2020-03-02.md [hunk 1]
```diff
@@ -299,6 +297,87 @@ This estimates are broadly consistent with those by other scientists which mostl
...
+
+Current case counts, deaths, and recoveries are being tracked in a number of places. These are updated as regularly as possible, but may not always be immediately up to date. One dashboards that we particularly like is from the JohnsCenter for Systems Science and Engineering:
+[Johns Hopkins CSSE Dashboard](https://gisanddata.maps.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6)
+
+#### How is COVID-19 spread?
+COVID-19 is a respiratory virus, which spread in 3 main ways:
+1. Between people who are in close contact with each other (within about 6 feet, or 2 metres), through small aerosolized particles that can be produced when a person talks, coughs, or sneezes. 
+2. In (relatively) larger droplets that are produced someone coughs or sneezes (these can travel over similar distances)
+3. Contact with surfaces or objects infected with the virus (also known as fomites). If a person touches an infected surface and then touches their eyes, nose, or mouth without washing their hands they may become sick by introducing the virus into their body.
```
Comment (nicfel) [inline narratives/ncov_sit-rep_2020-03-02.md]: Link/Source missing if possible

File: narratives/ncov_sit-rep_2020-03-02.md [hunk 2]
```diff
@@ -299,6 +297,87 @@ This estimates are broadly consistent with those by other scientists which mostl
...
+
+For COVID-19 there is evidence of transmission in clinically mild and asymptomatic cases, which means people who are infected with COVID-19 and able to transmit the virus to others, may not appear, or even feel, sick.
+
+#### Who is at the highest risk for COVID-19?
+Studies of patients in China have shown that most COVID-19 cases are adults, just 2.1% of 44,672 cases in China were in individuals below 20 years of age and just 1% of those are in children under 10. Approximately 80% of people experience mild illness, 14% experience severe disease, and 5% were critically ill. There is evidence of asymptomatic cases. 
+
+Based on case reports, individuals over 60 and those who have weakened immune systems or comorbidities like chronic lung disease, diabetes, and cardiovascular disease are at a higher risk for severe disease. Children do not seem to be at a high risk for either contracting COVID-19 or having severe disease as a result of an infection. A great graphic from [Jessie Bloom and his team](https://research.fhcrc.org/bloom/en.html that illustrates this point can be viewed [here] (https://raw.githubusercontent.com/jbloom/CoV_vs_flu_CFR/master/CFR-stats.jpg).
```
Comment (nicfel) [inline narratives/ncov_sit-rep_2020-03-02.md]: I would just quickly ask Jessie if we can use the figure

File: narratives/ncov_sit-rep_2020-03-02.md [hunk 3]
```diff
@@ -299,6 +297,87 @@ This estimates are broadly consistent with those by other scientists which mostl
...
+#### What can be done to prevent or reduce the spread of COVID-19?
+COVID-19 is a respiratory virus and nonpharmaceutical interventions are the most effective tool we currently have to prevent and slow the spread of COVID-19 at this time. There is not currently a vaccine or specific treatment for COVID-19. Some ways to implement nonpharmaceutical interventions include:
+
+- Practice [social distancing](https://en.wikipedia.org/wiki/Social_distancing), such as limiting attendance at events with large groups of people
+- Stay home, especially if you are feeling ill.
+- Implement good hand washing practices - it is extremely important to wash hands regularly  
+- Cover coughs and sneezes in your elbow or tissue
+- Avoid touching your eyes, nose, and mouth with unwashed hands
+- Disinfect frequently touched surfaces, such as doorknobs
+- Begint to take your temperature every day, if you develop a fever, stay home (if possible, separated from other people you live with) and call your doctor
```
Comment (nicfel) [inline narratives/ncov_sit-rep_2020-03-02.md]: typo

File: narratives/ncov_sit-rep_2020-03-02.md (hunk 1 above)
Comment (misjailcisin) [reply narratives/ncov_sit-rep_2020-03-02.md]: I *struggled* to find a good source that summed it up in one location, but I think we could link here: https://www.aappublications.org/content/35/1/1.2

reputable source and does go over all of them.

**Pull request merged**